cmd/expiration-mailer/main_test.go

```diff
@@ -386,6 +388,93 @@ func TestFindExpiringCertificates(t *testing.T) {
 	test.AssertEquals(t, len(testCtx.mc.Messages), 0)
 }
 
+func TestFindCertsAtCapacity(t *testing.T) {
+	testCtx := setup(t, []time.Duration{time.Hour * 24})
+
+	log.Clear()
+
+	// Add some expiring certificates and registrations
```

Comment:
Can this be factored out into a setup step? We already do it in TestFindExpiringCertificates.